src/main.rs

```diff
@@ -40,10 +52,18 @@ fn main() {
         8 => minifb::Scale::X8,
         _ => panic!("Supported scale: 1, 2, 4 or 8"),
     };
-    let mut window =
-        minifb::Window::new(format!("Gameboy - {}", rom_name).as_str(), SCREEN_W, SCREEN_H, option).unwrap();
+    let mut window_opt = None;
+    if !c_terminal {
```

Comment:
The judgment condition here should only be established when the user enters -t flag, the correct one should be `if c_terminal`, not `if !c_terminal`.